tests/components/application-connector/README.md

```diff
@@ -97,13 +106,13 @@ This will build the following images:
 - `{DOCKER_PUSH_REPOSITORY}/gateway-test:{DOCKER_TAG}`
 - `{DOCKER_PUSH_REPOSITORY}/mock-app:{DOCKER_TAG}`
 
-## Running
+### Running
 
 Tests can be run on any Kyma cluster with Application Gateway.
 
 Pipelines run the tests using the `test-gateway` target from the `Makefile`.
 
-### Deploy a Kyma cluster locally
+#### Deploy a Kyma cluster locally
 
 1. Provision a local Kubernetes cluster with k3d:
    ```sh
```

Comment:
It's in the OS docs, so we should probably use `Kyma OS (standalone mode)` and `SKR (Compass mode)` if we don't want to use **just** the modes.